awesomplete.css

```diff
@@ -1,4 +1,14 @@
 [hidden] { display: none; }
+[hiddena11y] {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
```

Comment:
It is very verbose indeed. Snook had done some [extensive research](http://snook.ca/archives/html_and_css/hiding-content-for-accessibility) and that was then adapted in the [HTML5 Boilerplate CSS](https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css#L131) which made this become the de facto standard to hide HTML only visually